### PR TITLE
Bump version to 2018.2.0

### PR DESCRIPTION
**Pull request by Adimote**

### CODE REVIEW

**Comment by Adimote** [comment]
@RealOrangeOne rebasing to master fixed the __VERSION__ bump issue

**Comment by Adimote** [comment]
(as in, rebasing to master has added the "2018.2.0" to the `__VERSION__` value)

File: debian/changelog [hunk 1]
```diff
@@ -3,3 +3,10 @@ robot-api (0) unstable; urgency=low
   * (No changelog)
 
  -- Andrew Barrett-Sprot <abarrettsprot@gmail.com>  Sun, 19 Nov 2017 10:47:00 +0000
+
+
+robot-api (2018.2.0) unstable; urgency=low
```
Comment (PeterJCLaw) [inline debian/changelog]: For some reason I was expecting the new version to be at the top. I'm not sure it matters which way around they go, but have we checked that this builds the debian package as we'd want?

Comment (Adimote) [reply debian/changelog]: ah looks like you might be right here

Comment (Adimote) [reply debian/changelog]: fixed in

**Pull request merged**